k8s/helm/values.yaml

```
@@ -17,10 +17,10 @@ component_operator_chart: "k8s/k8s-component-operator:latest"
 # Format: <namespace>/<name>: <version>
 components:
   # Use longhorn if your cluster has no storage provisioner.
-  #  k8s-longhorn:
-  #    version: latest
-  #    helmRepositoryNamespace: k8s
-  #    deployNamespace: longhorn-system
+  k8s-longhorn:
+    version: latest
+    helmRepositoryNamespace: k8s
+    deployNamespace: longhorn-system
```

```suggestion
  # k8s-longhorn:
  #   version: latest
  #   helmRepositoryNamespace: k8s
  #   deployNamespace: longhorn-system
```
I would revert that, to use longhorn only when it is needed